includes/functions.php

```diff
@@ -64,6 +64,17 @@ function can_replace( $post ) {
 		$replace = false;
 	}
 
+	/**
+	* Filter for Civil's can_replace logic.
```

Comment:
It is known that this is a filter, rather the title should describe what it does, rather than what it is.